core/src/main/java/com/graphhopper/routing/ch/CHParameters.java

```diff
@@ -12,7 +12,6 @@
     // node contraction, node-based
     public static final String EDGE_DIFFERENCE_WEIGHT = Parameters.CH.PREPARE + "node.edge_difference_weight";
     public static final String ORIGINAL_EDGE_COUNT_WEIGHT = Parameters.CH.PREPARE + "node.original_edge_count_weight";
-    public static final String CONTRACTED_NEIGHBORS_WEIGHT = Parameters.CH.PREPARE + "node.contracted_neighbors_weight";
```

Comment:
I removed this heuristic for now, might be worth to re-implement it later, but not sure really